src/App.tsx

```diff
@@ -1,15 +1,14 @@
 import { QueryClientProvider } from '@tanstack/react-query';
 import './App.css';
 import { Router } from './routing/router';
-import { ThemeProvider } from '@mui/material';
-import { materialUITheme } from './theme/materialUI';
+import { ThemeProvider } from './context/themeContext';
 import { SnackbarProvider } from 'notistack';
 import { queryClient } from './api/tanstack/queryClient';
 
 function App() {
   return (
     <>
-      <ThemeProvider theme={materialUITheme}>
+      <ThemeProvider>
         <QueryClientProvider client={queryClient}>
           <SnackbarProvider>
```

Comment:
check that snackbar changes to dark mode

Reply:
it does not